Adds Ross and Hacktoberfest HOWTO

zo0o0ot opened the pull request:
---
## 📢 Only PRs from existing Virtual Coffee members will be accepted.
---

## Type of Contribution

Check the item(s) that applies.

<!-- For example:
- [x] 🌱 Practice Open Source -->

- [x] 🌱 Practice Open Source
- [x] 📃 Repositories List

### If your type of contribution is Repositories List, check items in the repository checklist below that apply.

⚠️ _We only accept PRs where most of the items are checked_

- [x] Relevant name
- [x] Description
- [x] Relevant tags
- [x] README.md file
- [x] CONTRIBUTIONS.md/CONTRIBUTING.md file
- [x] Open source license
- [ ] Code of Conduct (COC)
- [x] Issue and Pull Request templates

## Description

<!-- Describe the changes that you've made in this PR. For example, "This PR adds <your-name> to the Preptember participants" or "This PR adds <repository-name> to the Repositories List." -->

This PR adds Ross Larson to the participants list, and also adds the [Hacktoberfest HOWTO repo](https://github.com/hacktoberfesthowto/howto-blog). This source code auto-deploys to a github pages site: https://hacktoberfesthowto.github.io/

adiati98 commented:
Hi @zo0o0ot,
Thank you for your contribution!

We have merged PRs before yours. Please resolve the conflicts first so we can merge your PR :)

Also, you submitted the [blog repo](https://github.com/hacktoberfesthowto/howto-blog) while you mentioned the [issue howto repo](https://github.com/zo0o0ot/hacktoberfest-issue-howto) in the PR description. We, maintainers, tend to grab what contributor mentioned in the PR desciption first to make review process faster.
So you might want to pay attention to this in the future :)

That said, which repo do you want to submit?

zo0o0ot commented:
Sorry about that.  You're right, I messed up my link in the PR.  

Please consider the [issue HOTWO repo](https://github.com/zo0o0ot/hacktoberfest-issue-howto) to be my official entry, not the original PR link.

The [issue HOTWO repo](https://github.com/zo0o0ot/hacktoberfest-issue-howto) was a precursor to the blog, and probably doesn't need future updates.

I've changed the link in the PR to point to the [HOWTO blog](https://github.com/hacktoberfesthowto/howto-blog) and I've updated the PR to avoid the merge conflicts.

Thanks for calling out my error.

adiati98 reviewed (approved):
@zo0o0ot thanks for resolving the conflict!
We are merging this PR with one take away.

You might want to raise an issue to the repo to add a Code of Conduct (COC).
Having a COC is very important in maintaining a safe and welcoming environmet for the contributors.
I saw that in other repo, they have a COC. You can suggest them to use the COC in the repo you submitted :)

zo0o0ot commented:
Great idea.  I've added the code of conduct to the howto-blog repo in https://github.com/hacktoberfesthowto/howto-blog/pull/39

adiati98 commented:
Awesome, @zo0o0ot!

![hell yeah GIF](https://media.giphy.com/media/dkGhBWE3SyzXW/giphy.gif)
